AsyncDisplayKit/ASViewController.mm

```diff
@@ -82,6 +83,13 @@ - (void)loadView
   _node.frame = frame;
   _node.autoresizingMask = autoresizingMask;
   self.view = view;
+  
+  // ensure that self.node has a valid trait collection before a subclass's implementation of viewDidLoad.
+  // Any subnodes added in viewDidLoad will then inherit the proper environment.
```

Comment:
OK, that makes perfect sense.  @rcancro You could slightly tweak the comment to clarify the main value of this is to ensure the trait collection is valid as early as possible rather than waiting for the later UIKit call, but that technically traitCollectionDidChange: will always be called at least once subsequently.